frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlDataOST.js

```diff
@@ -201,6 +202,15 @@ const controlDataOST = [
             required: true,
         },
     },
+    {
+        id: 'lbFloatingIP',
+        type: 'text',
+        name: 'creation.ocp.cluster.ost.lb.floating.ip',
+        placeholder: 'placeholder.creation.ocp.cluster.ost.lb.floating.ip',
+        tooltip: 'tooltip.creation.ocp.cluster.ost.lb.floating.ip',
+        active: '',
+        validation: VALIDATE_IP,
+    },
```

Comment:
We need to show this only when deploying 4.6 (or earlier) and show the `apiFloatingIP` field otherwise.

Reply:
👍🏾  Has been implemented